### DISCUSSION TITLE
Problems with exception 6 on ESP32_Relay X4_v1.1 - ESP32-WROOM

### DISCUSSION THREAD

**Discussion by balou1974** [arendst/Tasmota#17323] [Q&A]
### Discussed in https://github.com/arendst/Tasmota/discussions/17285

<div type='discussions-op-text'>

<sup>Originally posted by **balou1974** December  6, 2022</sup>
Hello,

I got a problem with a relay-card from aliexpress ESP32_Relay X4_v1.1 - it is working with default options (1 prog. button, 1 prog. led and 4x prog. relays) but as soon as I try to add a rf 433mhz receiver the whole thing reboots and clears the setting.

I tried different gpios and different power sources (the relay card can use 85-230V ac or 5 to 30V DC). I think this a flash or interrupt problem but kc868-a4 works with the same firmware (Tasmota 12.0.2.6 - self compiled and precompiled tested).

As soon as I connect the data-pin to the selected gpio it is over and the ESP32  is bringing an exception 6, reboots and clears the
settings.

GPIO0 Button, GPIO23 led, GPIO25 Relay 4, GPIO26 Relay 3, GPIO32 Relay 1, GPIO33 Relay 2

Tried different GPIOs for RFrecv GPIO 2, 27, 22, 35 nothing is working ... 

```
05:09:52.008 HTP: Web-Server aktiv bei tasmota-FFA12C-0300 mit IP-Adresse 192.168.178.126
05:09:52.012 RSL: INFO1 = {"Info1":{"Module":"ESP32-DevKit","Version":"12.2.0.6(tasmota)","FallbackTopic":"cmnd/DVES_FFA12C_fb/","GroupTopic":"cmnd/tasmotas/"}}
05:09:52.025 RSL: INFO2 = {"Info2":{"WebServerMode":"Admin","Hostname":"tasmota-FFA12C-0300","IPAddress":"192.168.178.126","IP6Global":"","IP6Local":"fe80::c249:efff:feff:a12c"}}
05:09:52.037 RSL: INFO3 = {"Info3":{"RestartReason":{"Exception":6,"Reason":"IntegerDivideByZero","EPC":"4008163e","EXCVADDR":"00000000","CallChain":["4008163b"]},"BootCount":42}}
05:09:56.979 QPC: Reset
```</div>

**Comment by Jason2866** [comment]
Moved to discussions, since issue template not filled out.

**Comment by Jason2866** [comment]
Please post your `user_config_override.h` and if you defined an new env this settings too.
Tried GPIO 19 to assign `RFrecv` no issue.

**Comment by soupman98** [comment]
Do you happen to have a pinout and the technical diagrams for this? I had them at one time and can't find them now.

**Reply by laczik** [reply]
@soupman98 Did you ever managed to locate a schematic PDF for the esp32_relay x4_v1.1 board?
